quill.js

```diff
@@ -1,13 +1,9 @@
 import Quill from './core';
 
-import Bold from './formats/bold';
-import Italic from './formats/italic';
-import Underline from './formats/underline';
-
 /**
- * This file defines which formatting modules should be included
- * into the final build of Quill.
- * Currently only Bold, Italic and Underline are allowed
+ * Here we export Quill without registering any formatting modules, because
```

Comment:
Might want to rewrite this comment a bit for clarity. After this change
> without registering any formatting modules

would be non-obvious for someone who hasn't worked on this project before and hasn't seen the module registration happen here in first place. You might want to explain how this file used to register the modules, linking to the `quill.reg.original.js`.

Also please make sure the "Disabling modules..." part makes it's way somewhere into the frontend where the registration happens, so the explanation of this quirk is also preserved.